etna/libs/ts2vec/ts2vec.py

```diff
@@ -93,6 +87,9 @@ def fit(self, train_data, lr=0.001, n_epochs=None, n_iters=None, verbose=False):
             n_epochs (Union[int, NoneType]): The number of epochs. When this reaches, the training stops.
             n_iters (Union[int, NoneType]): The number of iterations. When this reaches, the training stops. If both n_epochs and n_iters are not specified, a default setting would be used that sets n_iters to 200 for a dataset with size <= 100000, 600 otherwise.
             verbose (bool): Whether to print the training loss after each epoch.
+            device (str): The gpu used for training and inference.
+            batch_size (int): The batch size.
+            num_workers (int): How many subprocesses to use for data loading
```

Comment:
Is it useful?

Reply:
I think yes. Models are light and we can try to make training more efficient